main.star

```diff
@@ -195,11 +195,13 @@ def run(plan, args={}):
 
         first_cl_client = all_cl_client_contexts[0]
         first_client_beacon_name = first_cl_client.beacon_service_name
+        contract_owner, normal_user = genesis_constants.PRE_FUNDED_ACCOUNTS[6:8]
```

Comment:
@piwonskp - 
1. can you change the README to reflect this as well please?
2. payloads still get delivered with this configuration?

Reply:
1. Fixed
2. Yes